app/views/pets/show.html.erb

```diff
@@ -1,8 +1,8 @@
-<h1><%= @pet.name %></h1>
-<h3><%= @pet.age %> years old</h3>
-<h3><%= @pet.breed %></h3>
-<h3><%= @pet.adoptable %></h3>
-<h3><%= @pet.shelter_name %></h3>
+<h1>Name: <%= @pet.name %></h1>
+<h3>Age: <%= @pet.age %> years old</h3>
+<h3>Breed: <%= @pet.breed %></h3>
+<h3>Adoptable? <%= @pet.adoptable %></h3>
+<h3>Name of Shelter: <%= @pet.shelter_name %></h3>
 
 <p><%= link_to "Update #{@pet.name}", "pets/#{@pet.id}/edit" %></p>
 <p><%= link_to "Delete #{@pet.name}", "/pets/#{@pet.id}", method: :delete %></p>
```

Comment:
I would guess that this delete link doesn't actually work in the view. You need to use the turbo version to get to work.

```erb
<%= link_to "Delete #{@pet.name}", "/pets/#{@pet.id}", data: { turbo_method: :delete } %>
```